db/migrate/20240115083430_add_info_to_react.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# add columns
+class AddInfoToReact < ActiveRecord::Migration[7.1]
+  def change
+    change_table :reacts, bulk: true do |t|
```

Comment:
Please add unique for [:user_id, :micropost_id]
And add validate unique in model

Reply:
fixed